### PR TITLE
A few more variables to droplet_diag

### PR DESCRIPTION
**Pull request by drichte2**
Moved num100 and num1000, added a couple more stats in droplet_diag, and commented out some lines to clean up stdout

### CODE REVIEW

File: src/droplet.F [hunk 1]
```diff
@@ -519,16 +516,7 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
...
-    numvec(1) = num100
-    numvec(2) = num1000
 
-#ifdef MPI
-    if ( myid == 0 ) then
-       call mpi_reduce(mpi_in_place,numvec,2,mpi_int,mpi_sum,0,mpi_comm_world,ierr)
-    else
-       call mpi_reduce(numvec,numvec,2,mpi_int,mpi_sum,0,mpi_comm_world,ierr)
-    end if
-#endif
```
Comment (sjsprecious) [inline src/droplet.F:531]: So the update of `num100` and `num1000` is moved to `droplet_diag` now?

File: src/droplet.F [hunk 2]
```diff
@@ -3203,7 +3217,7 @@ subroutine droplet_injection(pdata,xf,yf,dt,time_since_inject,mtime,my_reintro)
                end do
                rad_init = rad_init*1.0e-6
                !DHR TEST, to make sure that droplets that destabilize numerics don't get produced
-               rad_init = min(rad_init,250.0e-6)
+               !rad_init = min(rad_init,250.0e-6)
```
Comment (sjsprecious) [inline src/droplet.F]: remove this line if it is no longer needed.

File: src/init3d.F [hunk 3]
```diff
@@ -407,7 +407,7 @@ subroutine init3d(xh,rxh,uh,ruh,xf,rxf,uf,ruf,yh,vh,rvh,yf,vf,rvf,  &
         if(dowr) write(outfile,*)
         if(dowr) write(outfile,*) '  Parcels ! '
         if(dowr) write(outfile,*) '  npvals,nparcelsLocal = ',npvals,nparcelsLocal
-        if(dowr) write(outfile,*) '  Initial parcel locations (x,y,z):'
+        !if(dowr) write(outfile,*) '  Initial parcel locations (x,y,z):'
```
Comment (sjsprecious) [inline src/init3d.F]: remove this line if it is no longer needed.

File: src/droplet.F (hunk 1 above)
Comment (drichte2) [reply src/droplet.F:531]: That's right. There is no longer any point doing it here. They are now part of the droplet_diag0 array

File: src/droplet.F (hunk 2 above)
Comment (drichte2) [reply src/droplet.F]: Fixed

File: src/init3d.F (hunk 3 above)
Comment (drichte2) [reply src/init3d.F]: Fixed

**Review by sjsprecious — APPROVED** [review]
Thanks @drichte2 for working on this PR and it looks good to me. The verification result on Derecho with `namelist_ASD.verify`namelist is slightly better than the result by using the latest `gpu-opt` branch.

ifort CPU versus nvhpc GPU
===========================

Metrics
-------
40 stat variables are identical.
43 stat variables show a mean relative difference > 1e-06
3 stat variables show a mean relative difference <= 1e-06

0th-order
---------
4 stat variables are identical.
8 stat variables show a mean relative difference > 1e-06
0 stat variables show a mean relative difference <= 1e-06

1st-order
---------
254 stat variables are identical.
2 stat variables show a mean relative difference > 1e-06
0 stat variables show a mean relative difference <= 1e-06

**Review by johnmauff — APPROVED** [review]
This looks good to me.

**Pull request merged**